apps/meteor/app/federation-v2/server/infrastructure/rocket-chat/callbacks/room.ts

```diff
@@ -0,0 +1,22 @@
+import { callbacks } from '../../../../../../lib/callbacks';
+import { IFederationBridge } from '../../../domain/IFederationBridge';
+
+class RocketChatRoomCallbacksHandler {
+	private bridge: IFederationBridge;
+
+	public injectBridgeInstance(bridgeInstance: IFederationBridge): void {
+		this.bridge = bridgeInstance;
+	}
+
+	public async afterLeaveRoom(rocketUser: any, rocketRoom: any): Promise<void> {
```

Comment:
I believe it would be better to have this logic inside the `RoomServiceSender`, so here we can just listen to the event itself, in this way all of our business logic will be in a single place, what do you think?